index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>課題①-3</title>
+  </head>
+  <body>
+    <script>
+      "use strict";
+
+      const heading = [
+        "========================",
+        "現在持っているタスクの一覧",
+        "========================",
+      ];
+      let tasks = ["掃除", "買い物", "散歩"];
+      let res = "";
+
+      function outputTaskList() {
+        for (let i = 0; i < heading.length; i++) {
+          console.log(heading[i]);
+        }
+        for (let j = 0; j < tasks.length; j++) {
+          console.log(j + "：" + tasks[j]);
+        }
+      }
+
+      outputTaskList();
+
+      while (res !== null) {
+        res = prompt("タスクを入力してください");
+
+        if (res === null) {
+        } else {
+          tasks.push(res);
+          alert("新しいタスクを追加しました。");
+          outputTaskList();
+        }
```

Comment:
以下のような記述でいいのではと思いました。

```suggestion
        if (res !== null) {
          tasks.push(res);
          alert("新しいタスクを追加しました。");
          outputTaskList();
        }
```